examples/step-63/doc/intro.dox

```diff
@@ -166,7 +164,10 @@ accordingly so that the ones further upstream are treated earlier
 (have larger indices). The influence of the ordering will be visible
 in the results section.
 
-Let us now briefly define the smoothers used in this tutorial. A Schwarz
+Let us now briefly define the smoothers used in this tutorial.
+For a more detailed introduction, we refer to
+@cite KanschatNotesIterative and the books @cite smith2004domain and @cite toselli2006domain.
```

Comment:
Does doxygen know to ignore the period at the end as far as the reference is concerned?

Reply:
yes, I compiled it locally and the reference shows up correctly.